### ISSUE TITLE
iOS and macOS are marked as misspelled

### ISSUE THREAD

**Issue by InnovativeInventor** [languagetool-org/languagetool#843]
iOS and macOS are marked as misspelled for some reason when I am on English.

**Comment by MikeUnwalla** [comment]
iOS is not correct typography. Refer to these pages:
https://www.apple.com/legal/intellectual-property/trademark/appletmlist.html
https://www.cisco.com/c/en/us/about/legal/trademarks.html

**Comment by InnovativeInventor** [comment]
I understand that iOS is not Apple's trademark, but I don't think it should be colored as misspelled when in fact, it is spelled correctly.

**Comment by MikeUnwalla** [comment]
@InnovativeInventor , Apple uses the trademark 'IOS' under licence from Cisco. On the Apple trademark page and on the Cisco trademark page, the word is fully capitalized.

Anyway, given that Apple cannot obey Apple's trademark guidelines, I added 'iOS' (https://github.com/languagetool-org/languagetool/commit/b33cb33ba1229d654327e35b715ad3b0fd6d79de). (I did not remove 'IOS', because that is a valid capitalization of Cisco's trademark.)
